tests/app/test_model.py

```diff
@@ -278,3 +290,94 @@ def test_user_can_use_webauthn_if_they_login_with_it(sample_user, auth_type, can
 
 def test_user_can_use_webauthn_if_in_notify_team(notify_service):
     assert notify_service.users[0].can_use_webauthn
+
+
+@pytest.mark.parametrize(('obj', 'return_val'), [
+    ({'a': None}, {}),
+    ({'b': 123}, {'b': 123}),
+    ({'c': None, 'd': 456}, {'d': 456}),
+    ({}, {})
+])
+def test_filter_null_value_fields(obj, return_val):
+    assert return_val == filter_null_value_fields(obj)
+
+
+def test_user_validate_mobile_number():
+    user = User()
+    with pytest.raises(ValueError):
+        user.validate_mobile_number('somekey', 'abcde')
+
+
+def test_user_password():
+    user = User()
+    with pytest.raises(AttributeError):
+        user.password()
+
+
+def test_annual_billing_serialize():
+    now = datetime.utcnow()
+    ab = AnnualBilling()
+    service = Service()
+    ab.service = service
+    ab.created_at = now
+    serialized = ab.serialize()
+    print(serialized)
+    expected_keys = ['id', 'free_sms_fragment_limit', 'service_id', 'financial_year_start',
+                     'created_at', 'updated_at', 'service']
+    for key in expected_keys:
+        assert key in serialized
+        serialized.pop(key)
+    assert serialized == {}
+
+
+def test_repr():
+
+    service = create_service()
+    sps = ServicePermission.query.all()
+    for sp in sps:
+        assert "has service permission" in sp.__repr__()
+
+    sgl = create_service_guest_list(service)
+    assert sgl.__repr__() == 'Recipient guest_list_user@digital.gov.uk of type: email'
```

Comment:
Is this leveraging an existing test fixture with a UK gov email?  Any chance we could change it easily?

Reply:
will look, although there is a separate "remove UK stuff from tests" ticket floating around.